packages/vue-components/src/Searchbar.vue

```diff
@@ -166,10 +166,120 @@ export default {
         }
       });
 
+      // using distance with adjacent transpositions
```

Comment:
Would be nice if you could explain what exactly this function does, since not everyone may know this algo.